pkg/apply/prune/grouping_errors.go

```diff
@@ -21,6 +24,13 @@ const multipleGroupingErrorStr = `Package has multiple grouping object templates
 The package should have one and only one grouping object template.
 `
 
+var groupingObjNamespaceError = `Attempting to apply namespace at the same time as the applied resources.
+
+Namespace: %s
+
+Currently, the namespace that applied resources are to be inserted into must exist before applying.
```

Comment:
I don't think we need to use the word Currently here. Let's just say that this is not supported.